rtsp_to_hls/lib/pipeline.ex

```diff
@@ -0,0 +1,136 @@
+defmodule Membrane.Demo.RtspToHls.Pipeline do
+  @moduledoc """
+  The pipeline, which converts the RTP stream to HLS.
+  """
+  use Membrane.Pipeline
+
+  require Logger
+
+  alias Membrane.Demo.RtspToHls.ConnectionManager
+
+  @impl true
+  def handle_init(options) do
+    Logger.debug("Source handle_init options: #{inspect(options)}")
+
+    connection_manager_spec = [
+      %{
+        id: "ConnectionManager",
+        start:
+          {ConnectionManager, :start_link,
+           [
+             [
+               stream_url: options[:stream_url],
```

Comment:
It is the first occurrence I've spotted, but you have used it multiple times below:
why not to use `options.stream_url` instead of `options[:stream_url]`? 
I know, that `options[:stream_url]` will return `nil` instead of raising an error when there isn't `:stream_url` key in `options` map - but is it the desired behaviour?

Reply:
done